File: Daily Exercise/删除排序数组中的重复项.py
class Solution:
    def removeDuplicates(self, nums:[int]) -> int:
        def solve(k):
            u = 0
            for x in nums:
                if u < k or nums[u - k] != x:
                    nums[u] = x
                    u += 1
            print(nums[0:u])
            return u
        
        return solve(2)


    def __init__(self):
      a = self.removeDuplicates([1,1,1,1,1,1,2,2,2,2,2,2,3])
      print(a)


if __name__ == "__main__":
    s = Solution()

File: Daily Exercise/删除排序数组中的重复项.cpp
class Solution{
    public:
    int removeuplicates(vector<int>& nums) {
        int n = nums.size();
        if (n <= 2) {
            return n;
        }

        int slow = 2, fast = 2;
        while (fast < n)
        {
            if(num[slow-2] != num[fast]) {
                nums[slow] = nums[fast];
                ++slow;
            }
            ++fast;
        }
        return slow;
    }
}